shell/common/persistent_cache.h

```diff
@@ -19,8 +23,17 @@ class PersistentCache : public GrContextOptions::PersistentCache {
 
   ~PersistentCache() override;
 
+  void AddWorkerTaskRunner(fml::RefPtr<fml::TaskRunner> task_runner);
+
+  void RemoveWorkerTaskRunner(fml::RefPtr<fml::TaskRunner> task_runner);
+
  private:
-  fml::UniqueFD cache_directory_;
+  std::shared_ptr<fml::UniqueFD> cache_directory_;
+  mutable std::mutex worker_task_runners_mutex_;
+  std::set<fml::RefPtr<fml::TaskRunner>> worker_task_runners_
```

Comment:
Does this need to be a multiset?

It might be possible for multiple Shell instances to share the same IO task runner instance

Reply:
Good call. Done.